libr/core/cmd_print.c

```diff
@@ -3273,26 +3283,30 @@ static int cmd_print_blocks(RCore *core, const char *input) {
 		}
 	}
 	switch (mode) {
-		case 'j':
-			pj_end (pj);
-			pj_end (pj);
-			r_cons_println (pj_string (pj));
-			pj_free (pj);
-			break;
-		case 'h': {
-			r_cons_printf ("\n%s\n", r_table_tofancystring (t));
-			r_table_free (t);
-			break;
+	case 'j':
+		pj_end (pj);
+		pj_end (pj);
+		r_cons_println (pj_string (pj));
+		break;
+	case 'h': {
+		r_cons_printf ("\n%s\n", r_table_tofancystring (t));
```

Comment:
Memory leak

Reply:
For what?

Reply:
The shared cleanup should deal with json.

Reply:
tofancystring() returns a string in the heap. you must free it

Reply:
Fixed.